Gopkg.toml

```diff
@@ -26,7 +26,8 @@
 
 [[constraint]]
   name = "github.com/cortexproject/cortex"
-  branch = "master"
```

Comment:
is it required ?

Reply:
The cortex branch which has retention changes got merged yesterday after opening this PR, so this change is not required now. Just need to update cortex in vendor.
Aim was to get retention working in single binary mode.